src/api/index.js

```diff
@@ -95,6 +95,13 @@ export const updateComment = ({
   const timestampId = `${new Date().getTime()}`;
   const userComment = userCommentText && userCommentText.trim();
 
+  if (userComment === '' || userComment === null) {
```

Comment:
Nope no reason - I haven't actually changed this, I just moved it and didn't consider it's implementation. Will change to `!userComment`.